docs/source/api_docs.rst

```diff
@@ -20,7 +20,7 @@ API Documentation
    Miscellaneous <pynwb.misc>
    Epoch <pynwb.epoch>
    PyNWB <pynwb>
-   Flexible Object-Relational Mapping <pynwb.form>
+   Flexible Object-Relational Mapping <hdmf>
```

Comment:
@ajtritt should ``Flexible Object-Relational Mapping `` be removed or renames here?

Reply:
@bendichter I think you may need to rebase to the current dev. I believe this line has been removed in the current dev. I think the same is true also for some of the other changes in the docs in the PR, where form has been changed to hdmf.